ui/uihandler_test.go

```diff
@@ -0,0 +1,295 @@
+// Copyright 2017 Istio Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package ui
```

Comment:
many of the functions in UI are hard to test

to add test to the ones that are testable, I'd move the function to http_utils first and add a test there

this being said, for this percentiles thing, you only need to put a simple test in Webtest and check for percentiles in the json output

Reply:
If you move the methods to http_utils. I can move the tests. Or you can tell me that which methods should I move to http_utils. I can move them.